### DISCUSSION TITLE
Is there a way to toggle off and on dropdown on arrow ?

### DISCUSSION THREAD

**Discussion by darkcris1** [orchidjs/tom-select#82] [Q&A] [answered]
I've been looking in to the documentation how to close dropdown using the arrow, but can't find it.

**Comment by oyejorge** [comment]
Which arrow are you referring to?

**Reply by darkcris1** [reply]
![IMG_20210511_210409](https://user-images.githubusercontent.com/65588059/117819842-858a2e80-b29c-11eb-8141-b43f091435e2.jpg)

**Reply by oyejorge** [reply]
Ok, wanted to make sure you weren't asking about the arrow key on keyboards.

At the moment, closing the dropdown via click events does not work. There's a mixture of mousdown and click events that, when fired, will open the dropdown. This caused the previous close-on-click functionality to only momentarily close the dropdown before it was immediately re-opened.

That said, I'll take another look to see if there's a sensible solution.

**Comment by oyejorge** [comment, marked as answer]
This has been fixed in v1.7

**Reply by darkcris1** [reply]
Great
